contributors/design-proposals/cluster-info-configmap.md

```diff
@@ -0,0 +1,25 @@
+# Persist cluster-wide data in ConfigMap
+
+## Abstract
+
+We want to allow cluster-wide data to be persisted for general purposes, currently for us (service catalog) is to focus on the ability to identify different clusters that access our service broker with a 'cluster-id'.
+
+## Motivation
+
+Currently there is no way for service catalog api server to identify the different clusters that access our service brokers. The goal of this design is to have a unify storage for cluster related information. The information should be retained and survive between component failures. 
```

Comment:
"cluster that accesses a service broker" is not well defined. Are we talking about pods within a cluster talking to the service-catalog apiserver? Are we talking about pods connection to services provisioned by the service broker?